models/linkages/R/write_restart.LINKAGES.R

```diff
@@ -226,9 +224,9 @@ write_restart.LINKAGES <- function(outdir, runid, start.time, stop.time, setting
     new.n.index <- c(new.n.index, rep(i, new.ntrees[i]))
   }
   
-  dbh.temp <- numeric(200)
-  iage.temp <- numeric(200)
-  nogro.temp <- numeric(200)
+  dbh.temp <- numeric(100)
```

Comment:
maybe assign this number `100` to a named variable to make it clear what it refers to, e.g. `n.trees`, `n.iters`

it would also help to change it once at one place in the future if it needs to be changing

Reply:
related: does the `200` below also need to be `100`?

```
    for (j in nl:nu) {
      dbh.temp[j] <- optimize(merit, c(1, 200), b_obs = b_obs[j], 
                              spp.biomass.params = spp.biomass.params)$minimum
}
```

Reply:
👍